src/server/routes/editor.js

```diff
@@ -207,7 +207,7 @@ async function getEditorActivity(editorId, startDate, Revision) {
 	const revisionsCount = _.countBy(revisionDates);
 
 	const allMonthsInInterval = eachMonthOfInterval({
-		end: Date.now(),
+		end: endDate,
```

Comment:
Addition of endDate is great.
I suggest one change that you can implement using TDD (test-driven development): first create a new test where you pass a string "bob" (or foo or bar or whatever :p) as the endDate.
I'm not sure what will happen with the current code, but you'll find out :)
Then, modify the test to expect it to throw a helpful error if the date is invalid.
Then, modify `getEditorActivity ` to act accordingly.
Finally, run the test again to ensure it's all working as expected.

It would be a good idea to do the same for startDate, now that I think of it.

You can use date-fns utilities to ensure the date is valid: https://date-fns.org/v2.12.0/docs/isValid#examples